### DISCUSSION TITLE
Issue Form template scrolling

### DISCUSSION THREAD

**Discussion by JennaBlack** [community/community#5902] [Projects and Issues]
When creating and editing an Issue Form template it is not possible to scroll from start to end and back to start in the UI. This also prevents previewing the form if you have scrolled to the bottom of the form. Have tried in multiple browsers and have experienced the same behaviour

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report

**Comment by Jay-o-Way** [comment]
Over a year old and this issue still isn't foxed. How's that possible?

**Comment by Jay-o-Way** [comment]
Ref: https://support.github.com/ticket/personal/0/1828438
